setup.sh

```diff
@@ -34,8 +35,13 @@ else
             fi
 
             conda install -y -c conda-forge mpi mpi4py petsc petsc4py
+            pip install tox
             # If complex number support is needed
             #conda install -y -c conda-forge mpi mpi4py "petsc=*=*complex*" "petsc4py=*=*complex*"
+
+            #  Environment variables
+            export PYTHONPATH="$CONDA_PREFIX/lib/python3.10/site-packages"
+            unset PETSC_DIR
```

Comment:
I don't understand why you unset `PETSC_DIR`?

Reply:
I reverted the changes in `test_petsc_installation`, but since in my shell the variable `PETSC_DIR` was already pointing to my local installation, I decided to unset it in `setup.sh`, such that the test can verify the PETSC installed in the virtual environment.